tests/integration/test_interaction.py

```diff
@@ -474,6 +541,15 @@ def test_item_negative_feedback_on_bad_input_with_keyboard(self, action_key):
     def test_final_feedback_and_reset_with_keyboard(self, action_key):
         self.parameterized_final_feedback_and_reset(self.items_map, self.feedback, action_key=action_key)
 
+    def test_alt_text_image(self):
```

Comment:
That might have been a mistake with my code, the intention was to make these tests entirely controlled by keyboard commands. Am I just missing a decorator above the method in `KeyboardInteractionTest`?